bin/prettier.js

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 "use strict";
+
```

Comment:
There must be a bug, it shouldn't automatically add empty spaces (even though this is what we want here)